sentim_client.go

```diff
@@ -0,0 +1,141 @@
+package jamie_chicken
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+type emotionResponse struct {
+	EmotionScore emotionScore `json:"EmotionScore"`
+	Emotion      string       `json:"Emotion"`
+}
+
+type emotionScore struct {
+	Anger          float32 `json:"Anger"`
+	Neutral        float32 `json:"Neutral"`
+	Others         float32 `json:"Others"`
+	Happy          float32 `json:"Happy"`
+	Joy            float32 `json:"Joy"`
+	Affection      float32 `json:"Affection"`
+	Sad            float32 `json:"Sad"`
+	Disappointment float32 `json:"Disappointment"`
+}
+
+type conversation struct {
+	Conversation []string `json:"conversation"`
+	Lang         string   `json:"lang"`
+	IgnoreFirst  bool     `json:"ignore_first,omitempty"`
+}
+
+type sentimClient struct {
+	auth_token string
+	base_url   string
+	logger     *logrus.Logger
+}
+
+func NewSentimClient() *sentimClient {
+	l := logrus.New()
+	l.SetOutput(os.Stdout)
+	l.SetLevel(logrus.DebugLevel)
+	return &sentimClient{
+		base_url: "https://api.sentimllc.com/",
+		logger:   l,
+	}
+}
+
+func (c *sentimClient) IsGritty(message string) bool {
+	return c.isEmotion("Anger", message)
+}
```

Comment:
Is `Anger` the only emotion we should be considering?

I think we need to pull in @DotNetGeekette

Reply:
Yeah, for grit that's probably right.

Reply:
Joy would probably be party-dinosaur

Reply:
we could get tricky with it too though, like
```
(anger > 0.5 || sad > 0.5 || disappointment > 0.5) && (joy < 0.2 || happy < 0.2)  == :party-dumpsterfire:
```